.github/workflows/crons.yaml

```diff
@@ -4,19 +4,25 @@ on:
   # NOTE: Schedule is only run on the latest commit of the default branch,
   # so we need to schedule Cron Jobs for all branches here
   schedule:
-    - cron: "1 0 * * 1-5"
+    # Run 129.0 nightly every weekday at 23:00
+    - cron: "0 23 * * 1-5"
+
+    # Run 128.0 nightly every weekday at 00:00
+    - cron: "0 0 * * 1-5"
 
 jobs:
   crons:
     runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
-        include:
+        jobs:
```

Comment:
You can keep `include` here, so that you avoid an extra layer "jobs"

Reply:
Thanks for the tip :)